core/src/main/java/com/graphhopper/routing/util/FerrySpeedCalculator.java

```diff
@@ -8,65 +8,58 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FerrySpeedCalculator.class);
     private final double speedFactor;
-    private final double unknownSpeed, longSpeed, shortSpeed, maxSpeed;
+    private final double unknownSpeed, maxSpeed;
 
-    public FerrySpeedCalculator(double speedFactor, double maxSpeed, double longSpeed, double shortSpeed, double unknownSpeed) {
+    public FerrySpeedCalculator(double speedFactor, double maxSpeed, double unknownSpeed) {
```

Comment:
`longSpeed` and `shortSpeed` were never used, because the distance was never missing (see below). At least when this code was used with our `OSMReader`. Also note that `longSpeed` and `shortSpeed` made no sense anyway for some encoders, because they exceeded `maxSpeed` (and this was not accounted for here).